hazelcast/src/main/java/com/hazelcast/jet/config/JobConfigArguments.java

```diff
@@ -40,6 +40,13 @@ public final class JobConfigArguments {
      */
     public static final String KEY_REQUIRED_PARTITIONS = "__sql.requiredPartitions";
 
+    /**
+     * The key under which the SQL engine (now) marks analyzed job.
+     * The reason for not having a separate flag in {@link JobConfig}
+     * is that we want to preserve Jet's independence from SQL.
+     */
+    public static final String IS_JOB_SUSPENDABLE = "__jet.jobIsSuspendable";
```

Comment:
suspendable = !analyzed

Reply:
Well, the idea is to split concepts: in terms of the Jet, we will have 'suspendable'  job, which is true by default. In SQL, query may be 'analyzed' to gain metrics. To prevent 'analyzed' query suspension and so on, we will use 'non-suspendable' job Jet concept. 
WDYT @k-jamroz ?

Reply:
previously the javadoc was:
>The key under which the SQL engine (now) marks analyzed job

"analyzed job" is NOT suspendable, which is negation of the name of the flag.

I see 2 reasonable possiblities:
1. can suspend job if IS_JOB_SUSPENDABLE is not present or IS_JOB_SUSPENDABLE = true
2. it is invalid to suspend job if IS_JOB_NOT_SUSPENDABLE = true

The former one uses less negations, so should be simpler.